earth_enterprise/src/fusion/autoingest/plugins.src/Database.src

```diff
@@ -263,7 +263,7 @@ DatabaseAssetImplD::HandleEditRequest(const DatabaseEditRequest &req)
     db->Modify(inputs, req.meta, dbConfig);
   } else {
     dbConfig.indexVersion = 1;
-    db = DatabaseFactory::Make(dbname, inputs, req.meta, dbConfig);
+    db = AssetFactory::Make<MutableDatabaseAssetD, DatabaseConfig>(dbname, inputs, req.meta, dbConfig);
```

Comment:
Very minor: I think you can get rid of the second template parameter here and several other places.